src/Utils/CategoryDropdown.js

```diff
@@ -38,6 +39,7 @@ function CategoryDropdown({ dropdownVendorCategories, name, withLabel, ariaLabel
       itemToString={toString}
       formatter={formatter}
       filter={filterItems}
+      id={id}
```

Comment:
what is the reason of that `id`? You can just change toSting
`const toString = useCallback((option) => `${name}-${option}`, [name]);`